### DISCUSSION TITLE
Drill down and filter box like select tab in every single chart on dashboard

### DISCUSSION THREAD

**Discussion by SakshiTharkude** [apache/superset#19706] [Ideas]
Clearly talking about APACHE SUPERSET the free source

Apache Superset compared to Power BI, Tableau has fulfilled many of the requirements about creating dashboards exporting the dashboard or chart to JSON and CSV.

One of the most important requirement  needed in Apache Superset according to me would be drilldown.

As this is a software which is under development , drilldown for dashboards would be best feature to add on to this software which will benefit this software a lot as it a free source unlike others.

The filter box : This visualization type is already present but it is only applicable for the dashboards and not for any specific chart 
If this filter option like a select tab would be added to the charts it would make this software more user friendly.

Users will increase, if these features are added.

**Comment by srinify** [comment]
Hey @SakshiTharkude moving to Github Discussions so we can keep Issues focused on bugs!

**Reply by SakshiTharkude** [reply]
Yes sure i can explain you in detail

**Reply by SakshiTharkude** [reply]
So basically drilldown report or drilldown through report.
A drill-down report allows the user to navigate the data from a comprehensive view of the data to a more detailed one
Drill downs are also used by SAP.
Drill Through Report
Like the drill-down report, a drill-through report supports users to navigate to a different layer of data by clicking a visualization element
